taitan.go

```diff
@@ -50,20 +52,23 @@ func getEnv(env string) string {
 }
 
 func getRoot() string {
+	if contentDir != "" {
+		return contentDir
+	}
 	content := getEnv("CONTENT_URL")
 	u, err := url.Parse(content)
 	if err != nil {
 		log.Fatalln("getContent: ", err)
 	}
 
-	// https://<token>@github.com/username/repo.git
-	u.User = url.User(getEnv("TOKEN"))
```

Comment:
A little worried about this, since it feels strange that it would be here without a reason.
But on the other hand, I can't really see any way that this would have any effect.

Reply:
Im pretty sure it's a sloppy copy-paste from `getContent` :p